src/net.c

```diff
@@ -118,6 +118,30 @@ int net__socket_accept(struct mosquitto_db *db, mosq_sock_t listensock)
 	struct request_info wrap_req;
 	char address[1024];
 #endif
+#if defined WITH_UNIX_SOCKETS && defined WITH_UDS_CP
+	struct fwd_con_ctx
+	{
+		uint8_t reserved;
```

Comment:
what's this for?

Reply:
since at least one byte needs to be sent along the way I just wanted to reflect that in the receiving iov buffer. That's all. At least for the receiving site iov can be NULL and length can be changed to zero ofcourse... Just wanted to keep consistency and annotate the possibility to forward other context related information.

Should I change that and recreate the PR with necessary Makefile update?